src/containers/profile/ProfileSections/ManageBlogs/ManageBlogs.jsx

```diff
@@ -1,7 +1,93 @@
 import React from "react";
+import { compose } from "redux";
+import { connect } from "react-redux";
+import { firestoreConnect } from "react-redux-firebase";
+import { Button, ListGroup } from "react-bootstrap";
+import BlogEditor from "../../../../containers/BlogEditor/BlogEditor";
+import BlogBox from "./BlogBox";
+import { approveBlog } from "../../../../actions/blogActions";
 
```

Comment:
can you please fix the directory paths, `src/containers/profile/ManageBlogs.jsx` like this